config/application-prod.yml

```diff
@@ -0,0 +1,9 @@
+tmc:
+  sandboxUrl: http://172.17.0.1:3001
+  notifyUrl: http://pingis.testmycode.io/submission-result
+
+security:
```

Comment:
Do the rest of the oauth2 config params still come from `config/application.yml`?